gatsby-node.js

```diff
@@ -4,60 +4,179 @@ var _regenerator = require("babel-runtime/regenerator");
 
 var _regenerator2 = _interopRequireDefault(_regenerator);
```

Comment:
Don't want to publish your built files! May want to add this to your gitignore!

Reply:
Just make sure you add an npmignore that only ignores the src directory!

Reply:
Yes, will add a npmignore file here